lib/tictactoe/console_io.ex

```diff
@@ -17,7 +17,22 @@ defmodule TicTacToe.ConsoleIO do
     |> output()
   end
 
-  def turn_message(%Player{name: name, type: :human}), do: output("#{name}'s turn:")
+  def player_menu(options) do
+    output("Who would you like to play against?")
+
+    options_with_number = Enum.with_index(options, 1)
```

Comment:
I've never seen `Enum.with_index`, but this is a really good solution. It's great that you can find and make extensive use of existing libraries rather than implement your own.